config/index.js

```diff
@@ -16,21 +16,21 @@ const sentryDsn = 'https://1f74b885d0c44549b57f307733d60351:dd736ff3ac994104ab66
 const DEFAULT_TIMEOUT = 2000;
 
 const overridableFields = [
-	'tokenKey', 'tokenPassword', 'testPackId', 'concurrency', // launcher automated
+	'tokenKey', 'tokenPassword', 'token', 'testPackId', 'concurrency', 'presets', // launcher automated
```

Comment:
should'nt be here `tokenId` instead of `token`?

Reply:
No. Please, refer to JS API test code snippet at https://mineus.atlassian.net/browse/SUIT-17374